src/ItkVtkViewProxy.js

```diff
@@ -133,7 +133,12 @@ function ItkVtkViewProxy(publicAPI, model) {
           xPosition: String(worldPosition[0]).substring(0, 4),
           yPosition: String(worldPosition[1]).substring(0, 4),
           zPosition: String(worldPosition[2]).substring(0, 4),
-          value,
+          value:
+            model.labelIndex === null
+              ? value
```

Comment:
After this change, `value` is no longer defined, causing a console error when there is no label map.

Reply:
Great catch  -- this should be `fusedValue`